### ISSUE TITLE
yarn is mandatory but script uses npm

### ISSUE THREAD

**Issue by mvbattan** [Wolox/wolmo-bootstrap-react-native#61]
## Summary

When I run
```
bash <(curl -s https://raw.githubusercontent.com/Wolox/wolmo-bootstrap-react-native/master/run.sh)
```
And I haven't installed `yarn`, it fails; but within the script, `npm` is being used instead.

**Comment by ignaciosantise** [comment]
@mvbattan two years later, we made the PR to solve this issue 😆 https://github.com/Wolox/wolmo-bootstrap-react-native/pull/160

**Comment by mvbattan** [comment]
@ignaciosantise Nice !
[![Nice](https://img.youtube.com/vi/3WAOxKOmR90/0.jpg)](https://www.youtube.com/watch?v=3WAOxKOmR90)

**Comment by sfernandez11** [comment]
Issue solved in #160
